IdentityCore/src/controllers/MSIDSilentController.m

```diff
@@ -114,7 +115,12 @@ - (void)acquireTokenWithRequest:(MSIDSilentTokenRequest *)request
     CONDITIONAL_START_EVENT(CONDITIONAL_SHARED_INSTANCE, self.requestParameters.telemetryRequestId, MSID_TELEMETRY_EVENT_API_EVENT);
     self.currentRequest = request;
     [request executeRequestWithCompletion:^(MSIDTokenResult *result, NSError *error)
-    {
+     {
+        MSIDAADRequestErrorHandler *handler = [[MSIDAADRequestErrorHandler alloc] init];
+        if (error && [handler shouldRetryNetworkingFailure:error.code]) {
```

Comment:
Can we also add an integration test for this change?

Reply:
Added integration test for identifying network failure